test/invoice_test.rb

```diff
@@ -130,27 +130,27 @@ def make_de_invoice_with_multiple_tax_rates
       )
       line_item = LineItem.new(
         name: 'Depfu Starter Plan',
-        quantity: 1,
+        quantity: 2,
```

Comment:
I changed the quantity here, because I wanted a quantity of 1 for the 7% item, to show the rounding problem